themes/local_theme_example/templates/search/home.phtml

```diff
@@ -28,7 +28,7 @@
       <?php endforeach; ?>
     </ul>
 
-    <ul class="navbar-nav navbar-right flip">
+    <ul class="nav navbar-nav">
```

Comment:
Same question here: I think flip is related to RTL support, so we should be sure to test that here.